frontend/src/http/lists.js

```diff
@@ -86,4 +86,33 @@ async function updateListName(id, name) {
         return err.error;
     };
 }
-export { getLists, getList, addItem, addList, setCompletion, updateListName };
+
+// delete functions!!!
+async function deleteItem(listId, id) {
+    try {
+        const res = await fetch(`/api/todos/${listId}/${id}`,
+            {
+                method: 'DELETE'
+            })
+        return res;
```

Comment:
This is not properly checking the response being successful or not - res.status would tell you if the response was success or not